### ISSUE TITLE
ST3 on Yosemite, Minification failed

### ISSUE THREAD

**Issue by oakley808** [tssajo/Minify#31]
I'm not sure if this is Yosemite related, but I did just upgrade.

Minification no longer works. 
In the ST console I see:
```
Unable to open /Users/moakley/Library/Application Support/Sublime Text 3/Packages/Default/Preferences.sublime-settings
Traceback (most recent call last):
  File "/Users/moakley/Library/Application Support/Sublime Text 3/Packages/Minify/Minify.py", line 83, in <lambda>
    sublime.set_timeout_async(lambda: self.do_action(), 0)
  File "/Users/moakley/Library/Application Support/Sublime Text 3/Packages/Minify/Minify.py", line 201, in do_action
    self.minify()
  File "/Users/moakley/Library/Application Support/Sublime Text 3/Packages/Minify/Minify.py", line 92, in minify
    view.run_command('save')
NameError: global name 'view' is not defined
```

**Comment by tssajo** [comment]
Thanks for the report. I'll look into it. It probably only happens on OSX, because the plugin works for me fine on Windows. I'll try it on OSX and I'll get back to you.

**Comment by tssajo** [comment]
@oakley808 Fixed in version 1.2.1 . Please update when 1.2.1 shows up in [Package Control](https://packagecontrol.io/packages/Minify). Report me if you encounter any issues with the plugin.
Thanks!
